### PR TITLE
Loki helm: Add Ingress labels for Loki gateway helm chart

### PR DESCRIPTION
**Pull request by tabramczyk**
Hello! I have small change suggestion. 
I think its too small to add documentation/changelog note, but if I should then just say and I can add. :)

**What this PR does / why we need it**: it adds possibility to set labels for Loki gateway Ingress.  

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [x] Documentation added
- [ ] Tests updated
- [x] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

**Review by JStickler — APPROVED** [review]
[docs team]  LGTM
Thank you very much for the contribution.

**Comment by tabramczyk** [comment]
Previous solution could generate malformed YAML so I fixed that.

**Comment by MichelHollands** [comment]
@tabramczyk Can you add a changelog entry and update the Helm chart version? An example can be found [here](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213). Also please rebase.

**Review by MichelHollands — APPROVED** [review]

File: production/helm/loki/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Entries should include a reference to the pull request that introduced the chang
 
 [//]: # (<AUTOMATED_UPDATES_LOCATOR> : do not remove this line. This locator is used by the CI pipeline to automatically create a changelog entry for each new Loki release. Add other chart versions and respective changelog entries bellow this line.)
 
+## 5.6.0
+- [ENHANCEMENT] Add `gateway.ingress.labels` to values and ingress-gateway in helm chart.
```
Comment (MichelHollands) [inline production/helm/loki/CHANGELOG.md:17]: Can you add a new line before and after this line? Just to keep it in the same format as the others.

**Comment by tabramczyk** [comment]
Thanks you for merging. :)

**Pull request merged**